src/app/Dashboard/CardStatus.tsx

```diff
@@ -0,0 +1,261 @@
+/* eslint-disable camelcase */
+import * as React from 'react';
+import {
+  Alert,
+  Card,
+  CardBody,
+  CardHeader,
+  Divider,
+  Flex,
+  FlexItem,
+  Grid,
+  GridItem,
+  Label,
+  NotificationDrawer,
+  NotificationDrawerBody,
+  NotificationDrawerGroup,
+  NotificationDrawerList,
+  NotificationDrawerListItem,
+  NotificationDrawerListItemBody,
+  NotificationDrawerListItemHeader,
+  Popover,
+  Title,
+} from '@patternfly/react-core';
+import { ExpandableRowContent, Table, Tbody, Td, Th, Thead, Tr } from '@patternfly/react-table';
+import BellIcon from '@patternfly/react-icons/dist/js/icons/bell-icon';
+import CheckCircleIcon from '@patternfly/react-icons/dist/js/icons/check-circle-icon';
+import ExclamationCircleIcon from '@patternfly/react-icons/dist/js/icons/exclamation-circle-icon';
+import ExclamationTriangleIcon from '@patternfly/react-icons/dist/js/icons/exclamation-triangle-icon';
+import global_success_color_100 from '@patternfly/react-tokens/dist/js/global_success_color_100';
+import global_danger_color_100 from '@patternfly/react-tokens/dist/js/global_danger_color_100';
```

Comment:
These will be the old v5 colors. Looks like these are just used to set icon colors, which can be updated to use `<Icon>` with the `status` prop for the color instead.